wallet-new/src/Cardano/Wallet/Kernel/Accounts.hs

```diff
@@ -14,10 +14,15 @@ import qualified Formatting as F
 import qualified Formatting.Buildable
 import           System.Random.MWC (GenIO, createSystemRandom, uniformR)
 
+import           Servant (err400)
```

Comment:
Maybe it's ugly to import all of the things required for the instances in each file containing errors.